src/rendering/RenderUtil.cc

```diff
@@ -2674,6 +2674,7 @@ void RenderUtilPrivate::HighlightNode(const rendering::NodePtr &_node)
     wireBoxVis->SetInheritScale(false);
     wireBoxVis->AddGeometry(wireBox);
     wireBoxVis->SetMaterial(white, false);
+    wireBoxVis->SetUserData("skip-visual-clone", static_cast<bool>(true));
```

Comment:
do you need to set this to other visuals like joint visuals, CoM, inertia, collision visuals?